zubhub_backend/zubhub/creators/utils.py

```diff
@@ -290,11 +290,12 @@ def send_notification(users: List[Creator], source: Creator, contexts,
             context.update({"phone": user.phone})
             sms_contexts.append(context)
 
+        template_name = get_notification_template_name(Setting.WEB, notification_type)
         message = render_to_string(
-            get_notification_template_name(Setting.WEB, notification_type),
+            template_name,
             context
         ).strip()
-        push_notification(user, source, notification_type, message, link)
+        push_notification(user, source, notification_type, message, link, template_name)
```

Comment:
I think we briefly talked about this today, but if grouped notifications end up requiring more context than just the ```sources``` we would have to pass ```context``` into ```push_notification``` and then it wouldn't really make sense to keep this web templating stuff here. Since we might end up moving it into ```push_notification``` anyway and it seems cleaner to isolate the web stuff there, it might be a good idea to just do that now.